ci/run_compute_sanitizer_test.sh

```diff
@@ -55,21 +55,26 @@ export LIBCUDF_MEMCHECK_ENABLED=1
 
 # Navigate to test installation directory
 TEST_DIR="${CONDA_PREFIX}/bin/gtests/libcudf"
-TEST_EXECUTABLE="$TEST_DIR/$TEST_NAME"
+TEST_EXECUTABLE="${TEST_DIR}/${TEST_NAME}"
 
-if [ ! -x "$TEST_EXECUTABLE" ]; then
-  rapids-logger "Error: Test executable $TEST_EXECUTABLE not found or not executable"
+if [ ! -x "${TEST_EXECUTABLE}" ]; then
+  rapids-logger "Error: Test executable ${TEST_EXECUTABLE} not found or not executable"
   exit 1
 fi
 
 # Run compute-sanitizer on the specified test
-compute-sanitizer --tool "$TOOL" --kernel-name-exclude kns=nvcomp "$TEST_EXECUTABLE" "$@"
+compute-sanitizer \
+  --tool "${TOOL_NAME}" \
+  --kernel-name-exclude kns=nvcomp \
+  --error-exitcode=1 \
```

Comment:
If the `--error-exitcode=1` works, then I'd like to see a run with the `GTEST_BRIEF=1` set again.

Reply:
Ok, so that output does not look helpful. 
The error is not going through the gtest so it is not outputing the test name around the `racecheck` failures.
Sorry, but it looks like removing `GTEST_BRIEF=1` is the right option for `racecheck` at least.